Sources/PodfilerKit/PodfileLockParser.swift

```diff
@@ -96,7 +94,10 @@ private func parse(pods: String) throws -> [Pod] {
         } else {
             transitives = []
         }
-        return Pod(name: name, version: Version(stringLiteral: version), dependencies: transitives)
+        guard let version = Version(string: versionString) else {
```

Comment:
Given that we do almost no validation of version strings, we don't want to crash if `Version` encounters something it doesn't support.